addons/payment_payulatam/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': 'PayuLatam Payment Acquirer',
+    'category': 'Payment Acquirer',
+    'summary': 'Payment Acquirer: PayuLatam Implementation',
+    'description': """Payulatam payment acquirer""",
+    'depends': ['payment'],
+    'data': [
+        'views/payment_views.xml',
+        'views/payment_payulatam_templates.xml',
+        'data/payment_acquirer_data.xml',
+    ],
+}
```

Comment:
I think you should add `'post_init_hook': 'create_missing_journal_for_acquirers',` like in the others `payment_x` modules